forc-plugins/forc-crypto/src/main.rs

```diff
@@ -16,7 +16,7 @@ mod keccak256;
 mod keys;
 mod sha256;
 
-const ABOUT: &str = "Forc plugin for hashing arbitrary data.";
+const ABOUT: &str = "Forc plugin for hashing arbitrary data";
```

Comment:
Clap can also parse the About from a doc comment, so you don't have to pass it in.

```rust 
/// Forc plugin for hashing arbitrary data.
#[derive(Debug, Parser)]
#[clap(
    name = "forc-crypto",
    after_help = help(),
    version
)]
pub enum Command {
```